app/models/opening_hour.rb

```diff
@@ -0,0 +1,20 @@
+# == Schema Information
+#
+# Table name: opening_hours
+#
+#  id          :bigint(8)        not null, primary key
+#  weekday     :string(255)
+#  date_from   :datetime
```

Comment:
sicher das `date_from` und `date_to` vom Type `datetime` sein sollen und nicht nur vom typ `string`?

Reply:
Weiß ich nicht, was meinst Du ? Eigentlich brauche ich hier nur ein Datum. ISt es möglich ein Datum ohne Uhrzeit zu haben oder muss es dann ein String sein, wenn das mein Wunsch ist.

Reply:
Wenn du als type `datetime` hast, dann ist immer auch eine Uhrzeit gespeichert/notwendig, ansonsten bietet sich der Type `date` an.
Meine Frage zielte aber eher auf die Daten ab, die da so kommen können/dürfen, ob die immer ein valides Datum sind? Oder ob wir auch "Januar 2019" oder "Herbst" akzeptieren werden/sollen/müssen.

Reply:
Die Daten die in den Converter kommen werden ja zunächst geparsed und alle anderen müssen sich an unsere vorgaben halten. Also würde ich sagen wir bekommen nur valide Daten.

Reply:
also dann vom type "date" statt "datetime" und "time" als notwendig

Reply:
Dann werde ich das aber dementsprechend anpassen in diesem PR datefelder werden zu date und Timefelder zum Typ time.